package-scripts/modules/ajexport.js

```diff
@@ -0,0 +1,64 @@
+// With thanks to elenterius on discord for troubleshooting
+// https://discord.com/channels/314078526104141834/1189404550986211329/1189517519262855229
+
+/* global Project, loadModelFile, AnimatedJava */
+
+const { readdirSync, readFileSync } = require('fs');
+
+export async function script() {
+  if (typeof AnimatedJava === 'undefined') {
+    throw new Error('Failed to load Animated Java plugin before CLI plugin');
+  }
+  const paths = getConfigPaths('./package-scripts/modules/config.json');
+  const modelDir = paths.ajmodelPath.concat('/');
+  console.log('Target paths: ', paths);
+  const files = readdirSync(modelDir).filter((file) =>
+    file.includes('ajmodel'),
+  );
+
+  for (const file of files) {
+    if (Project) {
+      Project.close();
+    }
+    const content = readFileSync(modelDir.concat(file), 'utf-8');
+    const name = file.split('/').pop();
+    const fileObj = {
+      path: file,
+      content: injectModelPackPaths(content, paths),
+      name: name,
```

Comment:
```suggestion
      name,
```

pls add `object-shorthand: 'error'` to `.eslintrc.yml`

https://eslint.org/docs/latest/rules/object-shorthand

Reply:
[4942163](https://github.com/TheAfroOfDoom/omega-flowey-minecraft-remastered/pull/64/commits/494216318489e524138553ed6562a2fe26133593)
and
[c6c9996](https://github.com/TheAfroOfDoom/omega-flowey-minecraft-remastered/pull/64/commits/c6c9996306c1d061b505ef06ed6b5918e01e1fc2)